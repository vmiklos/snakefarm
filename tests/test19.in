debug 1
out 1
time 14
load test19.map
addplayer 1
addsnake 1 1 2 1 1 0
17 0
18 0
19 0
endsnake
step
step
step
step
step
step
