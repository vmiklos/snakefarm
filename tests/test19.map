1;W;0;0;
2;W;0;1;
3;W;0;2;
4;W;0;3;
5;W;0;4;
6;W;0;5;
7;W;0;6;
8;W;0;7;
9;W;0;8;
10;W;0;9;

11;W;1;0;
12;0;2;11;
13;0;3;12;
14;0;4;13;
15;T;5;14;
16;T;6;15;
17;0;7;16;
18;0;8;17;
19;0;9;18;
20;W;10;19;

21;W;11;0;
22;W;12;21;
23;W;13;22;
24;W;14;23;
25;W;15;24;
26;W;16;25;
27;W;17;26;
28;W;18;27;
29;W;19;28;
30;W;20;29;
