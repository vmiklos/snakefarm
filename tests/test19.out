Event MapLoad test19.map
Event Step 1
StepEvent Snake_StoneBerry
End Step
Event Step 1
StepEvent Snake_StoneBerry
End Step
Event StoneStep 1
Event Step 1
End Step
Event StoneStep 1
Event StoneStep 1
Event Step 1
End Step
Event Step 1
End Step
Event Step 1
StepEvent Snake_Wall
StepEvent SnakeDie 1
End Step
